src/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
It's possible it doesn't; all I know is I ended up accidentally committing stuff into this directory while working on this PR.